docs/user-guide/index.rst

```diff
@@ -2,6 +2,18 @@
 
 .. include:: ModelAlignment.rsts
 
+.. toctree::
+   :maxdepth: 4
+   :titlesonly:
+
+   SFT.rst
+   RLHF.rst
+   SteerLM.rst
+   SteerLM2.rst
+   DPO.rst
+   SPIN.rst
+   DRaFTP.rst
+
 :ref:`Prerequisite Obtaining a Pre-Trained Model <prerequisite>`
    This section provides instructions on how to download pre-trained LLMs in .nemo format. The following section will use These base LLMs for further fine-tuning and alignment. 
```

Comment:
This section provides instructions on how to download pre-trained LLMs in .nemo format. The following section will use these base LLMs for further fine-tuning and alignment.